client/src/components/Create/Create.jsx

```diff
@@ -67,15 +67,13 @@ const Create = props => {
   let classes = styles();
   //Submission for the form
   const makeSubmission = () => {
-    Axios.post("./api/issue", {
+    Axios.post("http://localhost:8000/api/issue", {
       creatorId: props.creatorId || null,
       categoryName: category,
       headline: title,
       description: description,
       eventDate: date,
-      address: `${location}, Ausitn, Tx`,
-      lat: 0.0,
-      long: 0.0
+      address: `${location}, Ausitn, Tx`
```

Comment:
@Xenolithes 
Typo here. mispelled Austin and didnt capitalize X in TX
should be:
`${location}, Austin, TX`